index.html

```diff
@@ -45,6 +46,193 @@ <h1 class="heading__title fw-700">I'm Adeyemi.<br /> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+
+    <section class="tonic">
+        <img class="image is_desktop" src="assets/images/banners/desktop_tonic.svg" alt="tonic" />
+        <img class="image is_mobile" src="assets/images/banners/desktop_facebook.svg" alt="tonic" />
+
+        <div class="content">
+            <h1 class="title">Tonic</h1>
+
+            <p class="summary"><span>CANOPY</span><span class="dotted"></span><span class="color">Back End Dev</span><span class="dotted"></span><span class="color">2015</span></p>
+
+            <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            
+            <ul class="tagging ul flex">
+                <li>
+                    <button class="tag_button">html</button>
```

Comment:
- [ ] Good job implementing the `finish mobile version` milestone , however you still need to adjust some padding and margin to make sure that your design looks exactly as the Figma design 👍 

| Your design | Figma design |
| :---: | :---: |
| ![Capture](https://user-images.githubusercontent.com/80458199/235118544-b48e09e5-50b9-4821-bd06-2e8d1b65348d.PNG) | ![Capture`](https://user-images.githubusercontent.com/80458199/235118645-93bf9039-8cc5-4741-8d8b-b5f748acf71e.PNG) |